### PR TITLE
davidk/TSK-15176-throw-error-if-lib-categories-not-in-mask

### PR DESCRIPTION
**Pull request by DavidKorochik**
Taking care of the tests

### CODE REVIEW

File: components/library_automations/library/generator/actions/actions.go [hunk 1]
```diff
@@ -48,17 +48,21 @@ func GetActions(path string) (map[string]*types.Action, error) {
...
 	filteredActions := make(map[string]*types.Action)
 
 	for key, action := range actions {
-		if len(action.LibraryCategories) == 0 || action.Hidden || array.Contains(actionsBlackList, action.FullName) {
+		if action.Hidden || array.Contains(actionsBlackList, action.FullName) {
 			continue
 		}
 
+		if len(action.LibraryCategories) == 0 {
+			return nil, errors.New("There are no library categories in the mask file. Make sure the lib categories are located in the root of the mask file")
```
Comment (DanielDinsh) [inline components/library_automations/library/generator/actions/actions.go:61]:
```suggestion
			return nil, errors.Errorf("missing library categories in action %s", action.FullName)
```

**Review by DanielDinsh — COMMENTED** [review]
add a test case to this bug

**Comment by DavidKorochik** [comment]
> add a test case to this bug

@DanielDinsh Added a unit test :)

**Review by DanielDinsh — COMMENTED** [review]
I meant to add a test that will `require.NotNil(t, err)`

File: components/library_automations/library/generator/actions/actions_test.go [hunk 2]
```diff
@@ -54,11 +68,16 @@ func TestGetActions(t *testing.T) {
...
+		actionPath := filepath.Join(tmpPlaybooksDir, fmt.Sprintf("%s.yaml", actionData.Name))
 		err = ioutil.WriteFile(actionPath, actionYaml, 0644)
 		require.NoError(t, err)
 
-		actions, err := GetBlinkActions(tmpDir)
+		actions, err := GetBlinkActions(tmpPlaybooksDir)
+		if len(actionData.LibraryCategories) == 0 {
+			require.NotNil(t, err)
+			break
+		}
```
Comment (DanielDinsh) [inline components/library_automations/library/generator/actions/actions_test.go:79]: remove the `if`

**Review by DanielDinsh — APPROVED** [review]

**Pull request merged**